modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/PageMemoryDataRegion.java

```diff
@@ -33,4 +34,11 @@ public interface PageMemoryDataRegion {
      */
     @Nullable
     PageMemory pageMemory();
+
+    /**
+     * Returns page memory or throws an exception if not started.
+     */
+    default PageMemory requiredPageMemory() {
```

Comment:
Removed the added method and moved null-check to `pageMemory()` so that it is not `@Nullable` anymore (as from the analysis of the current code, no caller of `pageMemory()` tolerates null value)